ajax/libs/ionicons/package.json

```diff
@@ -23,7 +23,8 @@
     ]
   },
   "license": {
-    "name": "MIT"
+    "name": "MIT",
+    "url": "https://github.com/ionic-team/ionicons/blob/master/LICENSE"
```

Comment:
Let's modify from A to B to fit the CDNJS pattern.
- A
```
  "license": {
    "name": "MIT",
    "url": "https://github.com/ionic-team/ionicons/blob/master/LICENSE"
  },
```

- B
```
  "license": "MIT",
```

Thank you.